pages/common/git-sizer.md

```diff
@@ -0,0 +1,16 @@
+# git sizer
+
+> Computes various Git repository size metrics and alerts you to any that might cause problems or inconvenience.
+> Change to the directory containing your repository before running this command.
+
+- Report only statistics that have >0 level of concern:
+
+`git sizer`
+
+- Report all statistics:
+
+`git sizer -v`
+
+- See command line options:
```

Comment:
Perhaps `See additional help` might work better here? `See command line options` would imply to me that we aren't covering _any_ command like options in this tldr page, which is obviously not the case.

Reply:
Sure, I'll change it to `See additional options`